packages/scm/src/browser/scm-widget.tsx

```diff
@@ -149,6 +149,7 @@ export class ScmWidget extends BaseWidget implements StatefulWidget {
 
     protected onActivateRequest(msg: Message): void {
         super.onActivateRequest(msg);
+        this.refresh();
```

Comment:
I wonder should not we rather move it to `onAfterShow`, so it gets refreshed whenever the widget is revealed not focused?